src/redux/store.ts

```diff
@@ -1,12 +1,23 @@
 import { configureStore } from '@reduxjs/toolkit';
-import { authApi } from './slices/authSlice.ts';
+import { authApi } from './api/authApi.ts';
+import { chatApi } from '@/redux/api/chatApi.ts';
+import { userApi } from '@/redux/api/userApi.ts';
+import selectionReducer from '@/redux/slices/selectionSlice.ts';
+import authReducer from '@/redux/slices/authSlice.ts';
 
 export const store = configureStore({
   reducer: {
     [authApi.reducerPath]: authApi.reducer,
```

Comment:
Tohle není dobře. V celý appke by si měl využívat **jednu** instanci createApi. Případne jednu pro firebase a druhou pro nějaké jiné api. Samozrejme je dobre rozdelovat do vícero files jak to děláš, ale dá se to udělat přes injectEndpoints.